conf.pri.windows

```diff
@@ -46,15 +46,10 @@ DEFINES += BOOST_SYSTEM_STATIC_LINK
 #DEFINES += BOOST_SYSTEM_DYN_LINK
 # Boost 1.60+ defaults to Vista+ support. The define below enables XP support again.
 DEFINES += BOOST_USE_WINAPI_VERSION=0x0501
-# Enable if building against libtorrent 1.0.x (RC_1_0) (static linking)
```

Comment:
Does the comment means that we don't need it when building against libtorrent 1.1?

Reply:
Yes. Source code of Boost.Asio will be embedded to libtorrent-1.0 so we need one of these flags to use it. Currently we (and libtorrent) use header-only Boost.Asio library.